linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_admin_space_left_action/ansible/shared.yml

```diff
@@ -9,7 +9,7 @@
   lineinfile:
     dest: /etc/audit/auditd.conf
     line: "admin_space_left_action = {{ var_auditd_admin_space_left_action }}"
-    regexp: "^admin_space_left_action*"
+    regexp: "^admin_space_left_action\\s*=*"
```

Comment:
There is no need for double back slashes, and `=` is better without quantifiers.
Please change this in other expressions as well.

Reply:
@yuumasato yeah..... like it without the `\s`, but the double slashes are needed to pass ci.

Anyway, fixed without quantifiers.

Reply:
That is odd, there are other Ansible scripts with `\s` in `regexp`.